test/support/python_prism.py

```diff
@@ -0,0 +1,15 @@
+from lux.prism import Prism
+
+class SimplePrism(Prism):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def handle_signal(self, signal, context):
+        return {
+            'message': f'Hello, {signal.get("name", "prism")}!',
+        }
+
+
+def create_prism():
```

Comment:
Is this a function that would need to be implemented? Could we write this module as an example of a python prism in the way that others should write them? In other words, it should not have test code or "factory" stuff in it, and it should resemble a real prism that someone would write in python.

Also, we need to make sure that loading prisms with dependencies works out of the box.

Reply:
> Is this a function that would need to be implemented?

Unfortunately, yes.. For now we need a function to handle the Prism class instantiation.
Without this, I couldn't find a good place to define static values (e.g. id)

> Also, we need to make sure that loading prisms with dependencies works out of the box.

This will work like python module in `priv/python`, I'll add more test cases.

Reply:
This is amazing!